documentation/user_docs/docs/s_new.rst

```diff
@@ -0,0 +1,65 @@
+#####
+S new
```

Comment:
`S new` ?

Reply:
It was just one of the files in the Wiki with significant enough text to be included.